okteto/okteto
# refactor: move build model to pkg/build

## Motivation
- model pkg is a very large dependency which can be splited according to the command. Nowadays, the commands when they make use of the model import types and functions that are not required and with the possible problems that this can imply. With this PR the goal is to isolate the necessary model in the build command and reduce the dependency that this command currently imports. 

## Proposed changes
- move serialization of build section model into its own pkg `pkg/build`
- moves serialization tests
- creation of new pkg `pkg/utils` to avoid import cycles

## How to validate it
- running e2e test should be enough to know if the refactor is suitable
- all serialization logic needed only for `build` command is isolated in `pkg/build`
